x-pack/plugins/apm/server/routes/services/get_service_metadata_details.ts

```diff
@@ -104,12 +114,30 @@ export async function getServiceMetadataDetails({
             size: 10,
           },
         },
+        regions: {
```

Comment:
I think at this point we should revise our fetching strategy. Each added terms aggregation is another iteration of all the matching documents, and this adds three terms aggregations. Let's figure out what fields are available on what documents and see if we can build up a query that returns a single document that has them all.

Reply:
As discussed, this request is made only when a service metadata icon is clicked and it can be addressed in another PR. Raised https://github.com/elastic/kibana/issues/123957